src/components/carousel/carousel.js

```diff
@@ -564,29 +581,29 @@ export const BCarousel = /*#__PURE__*/ Vue.extend({
     const indicators = h(
       'ol',
       {
-        class: ['carousel-indicators'],
+        staticClass: suffixClass(CLASS_NAME_CAROUSEL, 'indicators'),
         directives: [
           { name: 'show', rawName: 'v-show', value: this.indicators, expression: 'indicators' }
         ],
         attrs: {
-          id: this.safeId('__BV_indicators_'),
+          id: this.safeId('_BV_indicators_'),
           'aria-hidden': this.indicators ? 'false' : 'true',
           'aria-label': this.labelIndicators,
-          'aria-owns': this.safeId('__BV_inner_')
+          'aria-owns': this.safeId('_BV_inner_')
         }
       },
       this.slides.map((slide, n) => {
         return h('li', {
           key: `slide_${n}`,
-          class: { active: n === this.index },
+          class: { [CLASS_NAME_ACTIVE]: n === this.index },
```

Comment:
Perhaps switch from Dynamic property names (when the property name is not a computation) to array format:

```js
class: [n === this.index ? CLASS_NAME_ACTIVE : null]
```

As dynamic object properties transpile out to a bit longer code (except if browserlistrc excludes older browsers that do not support dynamic property names)